engines/stark/stark.cpp

```diff
@@ -175,6 +175,8 @@ void StarkEngine::setStartupLocation() {
 }
 
 void StarkEngine::mainLoop() {
+	// Location change scripts require >0 loop duration
+	_global->setMillisecondsPerGameloop(10);
```

Comment:
Set a (small) default value from the frame duration when the computed value is zero?